lib/validate.rb

```diff
@@ -0,0 +1,20 @@
+class Validate
+
+    def validate_input(input, board)
+        if valid_input?(input) && free_spot?(input, board)
+            return true
+        else
+            return false
+        end 
+    end
+
+    private
+
+    def valid_input?(input)
```

Comment:
Thinking about the naming of these methods...

I'm wondering if `valid_input?` could be changed to `valid_integer?` or something along those lines, to be more specific, and `validate_input` could be called `valid_free_spot?` or whatever you think a good name for that would be. `valid_free_spot?` indicates to me that it returns a boolean. open to disagreements or discussion!

Reply:
This is a good call out, I like both of these suggestions! Since my original thought was that `validate_input` would throw an error instead of returning false, I wasn't thinking of it as a boolean. `valid_integer` is also more descriptive so I prefer it as well!